e2e-tests/kubernetes-headlamp-ci.yaml

```diff
@@ -10,34 +23,66 @@ spec:
     - port: 80
       targetPort: 4466
   selector:
-    k8s-app: headlamp
+    app.kubernetes.io/name: headlamp
+    app.kubernetes.io/instance: headlamp
 ---
-kind: Deployment
+# Source: headlamp/templates/deployment.yaml
 apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: headlamp
   namespace: kube-system
+  labels:
+    app.kubernetes.io/name: headlamp
+    app.kubernetes.io/instance: headlamp
 spec:
   replicas: 1
   selector:
     matchLabels:
-      k8s-app: headlamp
+      app.kubernetes.io/name: headlamp
+      app.kubernetes.io/instance: headlamp
   template:
     metadata:
       labels:
-        k8s-app: headlamp
+        app.kubernetes.io/name: headlamp
+        app.kubernetes.io/instance: headlamp
     spec:
+      initContainers:
+      - command:
+        - /bin/sh
+        - -c
+        - mkdir -p /build/plugins && cp -r /plugins/* /build/plugins/
+        image: ghcr.io/yolossn/headlamp-plugins:latest
+        imagePullPolicy: Always
+        name: headlamp-plugins
+        volumeMounts:
+        - mountPath: /build/plugins
+          name: headlamp-plugins
       containers:
       - name: headlamp
-        imagePullPolicy: Never
-        image: ghcr.io/headlamp-k8s/headlamp:latest
+        securityContext:
+          privileged: false
+          runAsGroup: 101
+          runAsNonRoot: true
+          runAsUser: 100
+        image: "ghcr.io/headlamp-k8s/headlamp:latest"
+        imagePullPolicy: IfNotPresent
```

Comment:
If we use `Always` it won't use the built image, it will download the image from the link. I think it should be `Never`, then it will use the one that we recently built

Reply:
Awesome, thanks.